modules/test/playwright/pages/roles-admin-web/blogs/BlogsPage.ts

```diff
@@ -15,11 +17,16 @@ export class BlogsPage {
 		this.blogContainer = page.locator(
 			'#portlet_com_liferay_blogs_web_portlet_BlogsPortlet'
 		);
-		this.blogTitle = (title) =>
-			this.blogContainer.getByRole('link', {name: title});
+		this.blogTitle = (title: string) => page.getByText(`${title}`);
```

Comment:
you don't need `${title}` as you are not creating a string, title it's already a variable
```page.getByText(title);```